internal/ansible/proxy/cache_response.go

```diff
@@ -86,7 +85,6 @@ func (c *cacheResponseHandler) ServeHTTP(w http.ResponseWriter, req *http.Reques
 		}
 
 		if c.skipCacheLookup(r, k, req) {
-			log.Info("Skipping cache lookup", "resource", r)
```

Comment:
imo we should not remove the log. We need to change its level.

Reply:
I tried but seems `log` does not support debug level

Reply:
You can add debug logging like
```go
log.V(1).Info("Skipping cache lookup", "resource", r)
```
That `1` indicates a log message that only gets logged when a [development logger](https://pkg.go.dev/github.com/kubernetes-sigs/controller-runtime/pkg/log/zap#Logger) is used.

Reply:
Thanks. I have put to V(2) to align with other lines in the same function. If 1 is really you want. I can make the change again.

Reply:
I think 2 is ok, although there is a mix of 1's and 2's that should probably be made 1's (helm-operator only uses 1's).